components/layout_2020/flow/root.rs

```diff
@@ -38,6 +40,10 @@ pub struct BoxTree {
     /// Contains typically exactly one block-level box, which was generated by the root element.
     /// There may be zero if that element has `display: none`.
     root: BlockFormattingContext,
+
+    /// https://drafts.csswg.org/css-backgrounds/#special-backgrounds
+    #[serde(skip)]
```

Comment:
Is there some reason to not simply implement `Serialize` and `Deserialize` on `CanvasBackground`?

Reply:
`ComputedValues` doesn’t implement `Serialize`, but I can push the `skip` attribute down to that field of `CanvasBackground`.